joint_limits:

  wheel_left_joint:
    has_position_limits: false
    min_position: -1.0
    max_position: 1.5708
    has_velocity_limits: true
    max_velocity: 1000.0 #rad/s
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
    min_effort: -1.0

  wheel_right_joint:
    has_position_limits: false
    min_position: -1.0
    max_position: 1.5708
    has_velocity_limits: true
    max_velocity: 1000.0
    has_acceleration_limits: false
    max_acceleration: 0.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
    min_effort: -1.0
